### ISSUE TITLE
A11y: Modals lack focus trap and restore focus on close

### ISSUE THREAD

**Issue by EntchenEric** [EntchenEric/jellyfin-auto-groupings#644]
`showModal` focuses first element but Escape closes without returning focus.

**File:** `ui.js`

**Comment by EntchenEric** [comment]
Resolved in #1004 (merged to development).
